ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -3190,3 +3190,16 @@ literals:
   - ACTIVE
   - INACTIVE
 
+- mixed_air_damper_open_command:
```

Comment:
Is the mixed air damper truly binary? Or can it modulate? Is there a single field that can represent the damper position (I.e., `mixed_air_damper_command` is OPEN/CLOSED)?